### DISCUSSION TITLE
Support for the Delete feature in version 7

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Knowlg/Community#208] [General Discussions]
Hi Team,

We would like to know how and from where I can get the delete functionality patch. 

cc: @pallakartheekreddy @krgauraw @@rajeevsathish @mohitga @vinukumar-vs @ashokreddy1208 @swayangjit

**Comment by pallakartheekreddy** [comment]
@siddhishinde0723 we have already provided details in below thread which was created by you. Please check below
https://github.com/orgs/Sunbird-Knowlg/discussions/181

**Reply by siddhishinde0723** [reply]
@pallakartheekreddy  thanks you i missed out the ticket.
One question where and how can i get a code patch?
